.eslintrc.json

```
@@ -0,0 +1,15 @@
+{
+	"env": {
+		"es6": true,
+		"node": true
+	},
+	"parserOptions": {
+		"sourceType": "module",
+		"ecmaVersion": 2019
+	},
+	"plugins": ["prettier"],
+	"extends": ["plugin:prettier/recommended"],
```

Extend the default eslint rules
```suggestion
	"extends": ["eslint:recommended", "prettier", "plugin:prettier/recommended"],
```

---

You use jest and react-testing-library. You can add:

- [eslint-plugin-testing-library](https://github.com/testing-library/eslint-plugin-testing-library#recommended)
- [eslint-plugin-jest](https://github.com/jest-community/eslint-plugin-jest#recommended)

ie `extends: ['plugin:jest/recommended', 'plugin:jest-dom/recommended', 'plugin:jest/style'],`